sfofferchrome/payload.js

```diff
@@ -1,13 +1,8 @@
 // send the page title as a chrome message
 
-var s = document.createElement("script");
-s.src="https://code.jquery.com/jquery-3.4.1.js";
-s.onload = function(e){ /* now that its loaded, do something */ }; 
-document.head.appendChild(s); 
-$(document).ready(function(){
-

```

Comment:
Why are lines 3-8 removed?

Reply:
I mentioned in Slack why I removed these lines.

I can't say 100% why they were breaking the script, but I believe it was because we were trying to unsuccessfully load jQuery after it was already being successfully loaded by the extension's manifest.

By having the correct copy of jquery in the content_scripts portion of the manifest, Chrome handles injecting it into the page for us and so we don't need to do it in the payload.

I'm not sure which document it was referencing. In both lines 6 and 7, the document referenced is the html concept of the document https://www.w3schools.com/jsref/dom_obj_document.asp. This makes me think the code was being run too early and the document wasn't fully loaded.